docker/manage

```diff
@@ -178,7 +178,7 @@ configureEnvironment() {
   # controller
   export IDENTITY_SERVER_URL="${IDENTITY_SERVER_URL:-http://localhost:5000}"
   export IDENTITY_SERVER_API_KEY="controller-api-key"
-  export IDENTITY_SERVER_WEB_HOOK_URL=${IDENTITY_SERVER_URL}
+  export IDENTITY_SERVER_WEB_HOOK_URL=${IDENTITY_SERVER_WEB_HOOK_URL:-http://$DOCKERHOST:5000}
```

Comment:
This is the url used buy the agent to deliver webhooks, so it has to be on the internal docker network.